trollflow2/launcher.py

```diff
@@ -145,14 +144,17 @@ def process(msg, prod_list):
                     cwrk = wrk.copy()
                     cwrk.pop('fun')(job, **cwrk)
             except AbortProcessing as err:
-                LOG.info(str(err))
+                LOG.info("Worker %s failed: %s", str(wrk['fun']), str(err))
     except Exception:
         LOG.exception("Process crashed")
         if "crash_handlers" in config:
             trace = traceback.format_exc()
             for hand in config['crash_handlers']['handlers']:
                 hand['fun'](config['crash_handlers']['config'], trace)
 
+    # Remove config so all remaining references are removed
+    del config
```

Comment:
This will fail if the call to `open` crashed

Reply:
Ah, good catch. I'll wrap it in try/except.